rhc-ose-ansible/inventory/ose-provision

```diff
@@ -9,9 +8,15 @@ openstack_key_name=""
 #openshift_storage_disk_volume=/dev/vdb
 #openstack_flavor_name="m1.medium"
 #openstack_image_name="ose3_1-base"
-#openshift_openstack_master_storage_size: 10
-#openshift_openstack_node_storage_size: 10
-
+#openshift_openstack_master_storage_size=10
+#openshift_openstack_node_storage_size=10
+#env_id=
+inv_cluster_method=native
```

Comment:
cc: @detiber, want to get your opinion on this. We're basically using this playbook & role as a "decoupler" from our playbooks and the openshift-ansible stuff.